javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,41 @@
+/*console.log("Script loaded");
+
+const products = getAvailableProducts();
+console.log(products);
+*/
+
+
+
+
+
+const products = getAvailableProducts();
+const ul = document.getElementById("products_list");
+const input = document.getElementById("filter_products");
+const value = input.value;
+
+input.addEventListener('input', filtered_products);
```

Comment:
The problem here is that console gives an error: "main.js:16 Uncaught ReferenceError: Cannot access 'filtered_products' before initialization"

And from documentation:
![image](https://user-images.githubusercontent.com/31403464/130363597-3a2d2ccc-5949-4e64-abd1-5abbedfcd4ef.png)
 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function#function_expression_hoisting

So basically you need to declare functions fiirst, and then call it in addEventListener